packages/grid/_modules_/grid/hooks/features/density/useDensity.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+import { useLogger } from '../../utils/useLogger';
+import { ApiRef } from '../../../models/api/apiRef';
+import { useApiMethod } from '../../root/useApiMethod';
+import { useGridSelector } from '../core/useGridSelector';
+import { useGridState } from '../core/useGridState';
+import { DensityApi } from '../../../models/api/densityApi';
+import { Density, DensityTypes } from '../../../models';
+import { optionsSelector } from '../../utils/useOptionsProp';
+import { DensityState } from './densityState';
+
+export const SIZE_FACTOR_SMALL = 0.7;
+export const SIZE_FACTOR_LARGE = 1.3;
```

Comment:
Something like this?

```suggestion
export const COMPACT_DENSITY_FACTOR = 0.7;
export const COMFORTABLE_DENSITY_FACTOR = 1.3;
```

Reply:
done